CHANGELOG.md

```diff
@@ -18,6 +18,8 @@
   | [#283](https://github.com/JetBrains/bazel-bsp/pull/283)
 - Don't use `Label.toString` now that it's no longer overriden.
   | [#282](https://github.com/JetBrains/bazel-bsp/pull/282)
+- Fix transitive target failure check in bloop export
```

Comment:
please add it in the `unreleased` section

Reply:
oops, done